### PR TITLE
Improvements to example of reprojection of HMI onto AIA

### PR DESCRIPTION
**Pull request by ayshih**
- Removed downloading, and instead uses files from the sample data set (see #4836)
- Explained the HMI color map and set the range correctly
- Fixed plot titles

### CODE REVIEW

**Review by Cadair — APPROVED** [review]

**Review by nabobalis — APPROVED** [review]

**Review by hayesla — APPROVED** [review]

File: examples/map_transformations/reprojection_align_aia_hmi.py [hunk 1]
```diff
@@ -14,49 +14,25 @@
...
+# HMI map, we use the special HMI color map, which expects the plotted
+# range to be -1500 to 1500.
 
-map_aia, map_hmi = [m.resample((1024, 1024)*u.pix) for m in sunpy.map.Map(sorted(files))]
-# Why do we have to do this?
+map_aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+map_hmi = sunpy.map.Map(sunpy.data.sample.HMI_LOS_IMAGE)
 map_hmi.plot_settings['cmap'] = "hmimag"
-map_hmi.plot_settings['norm'] = plt.Normalize(-2000, 2000)
+map_hmi.plot_settings['norm'] = plt.Normalize(-1500, 1500)
```
Comment (dstansby) [inline examples/map_transformations/reprojection_align_aia_hmi.py:28]: Should we be setting these in the map source?

Comment (Cadair) [reply examples/map_transformations/reprojection_align_aia_hmi.py:28]: It's only needed if you use the hmimag colourmap which is scaled to these values aiui.

Comment (dstansby) [reply examples/map_transformations/reprojection_align_aia_hmi.py:28]: I guess my question was should we be defaulting to the `hmimag` colormap?

Comment (ayshih) [reply examples/map_transformations/reprojection_align_aia_hmi.py:28]: I don't believe it should be the default.  I plot HMI data only very rarely, but when I do, I always use grayscale, and that's how I usually see it.  Maybe I just don't run in the circles where use of the `hmimag` colormap is commonplace.  I also wonder if the colormap is appropriate for all types of HMI files that could be opened by `HMIMap`.

**Pull request merged**